### ISSUE TITLE
Support for BigInt type

### ISSUE THREAD

**Issue by ichi404gh** [SkeLLLa/node-object-hash#39]
Hashing fails when there is BigInt type.

```node
> const Hasher = require('node-object-hash')
> const hasher = Hasher()
> hasher.hash(1)
'6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4b'
> hasher.hash(1n)
Uncaught TypeError: stringifiers[typeGuess_1.guessType(...)] is not a function
    at objectToString (***/node_modules/node-object-hash/dist/objectSorter.js:135:56)
    at Object.hashObject [as hash] (***/node_modules/node-object-hash/dist/hasher.js:33:22)

```

**Comment by SkeLLLa** [comment]
Hi. Yes, Bigint was introduced not so long ago, so there's no support currently.

Would you like to send a PR for this case?

**Comment by SkeLLLa** [comment]
Implemented in #40. Released in v2.2.0.
